susemanager/src/mgr_bootstrap_data.py

```diff
@@ -447,51 +107,17 @@
     "gpgsm",
     "gpg-wks-client",
     "gpg-wks-server",
-    "iso-codes",
     "libassuan0",
     "libksba8",
     "libldap-2.4-2",
     "libldap-common",
-    "libnorm1",
-    "libnpth0",
-    "libpgm-5.2-0",
     "libsasl2-2",
     "libsasl2-modules",
     "libsasl2-modules-db",
     "libsodium23",
-    "libyaml-0-2",
-    "libzmq5",
-    "lsb-release",
     "pinentry-curses",
-    "python3-apt",
-    "python3-certifi",
-    "python3-chardet",
-    "python3-croniter",
-    "python3-crypto",
-    "python3-pycryptodome",
-    "python3-dateutil",
-    "python3-distro",
-    "python3-idna",
-    "python3-jinja2",
-    "python3-markupsafe",
-    "python3-msgpack",
-    "python3-pkg-resources",
-    "python3-psutil",
-    "python3-requests",
-    "python3-six",
-    "python3-systemd",
-    "python3-tz",
-    "python3-urllib3",
-    "python3-yaml",
-    "python3-zmq",
-    "python-apt-common",
-    "salt-common",
-    "salt-minion",
-    "gnupg",
+    # end of gnupg dependencies
     "venv-salt-minion",
-    "python3-gnupg",
-    "python3-looseversion",
-    "python3-jmespath",
 ]
 
 PKGLISTDEBIAN11 = [
```

Comment:
not sure what to do here. Debian 11 is in our grace period before EOL.

Reply:
Then let's keep it for now